app/templates/start.html

```diff
@@ -65,8 +35,7 @@ <h1 class="ons-u-fs-xxl ons-u-mt-l">{{_('Start study')}}</h1>
         onsUACInput({
             'id': 'uac',
             'name': 'uac',
-            'postTextboxLinkText': _('Where to find your access code'),
-            'postTextboxLinkUrl': where_code_link,
+            'postTextboxLinkText': _('Your personal access code can be found in the letter or email you received which linked you to this survey.'),
```

Comment:
Should this still be a link? When clicked on it just links back to the same page

Reply:
It shouldn't be a link but there's another ticket to deal with it as it being a link is part of the Design System component used on the Start page and we have no control over it